public/src/plotArea.js

```diff
@@ -80,6 +80,15 @@ const plotArea = {
     resume() {
         this.autoScroll = true;
     },
+    // pause autoscroll
+    pause() {
+        this.autoScroll = false;
+        // added below lines to avoid bugs
+        plotArea.mouseDown = false;
+        var time = new Date().getTime() - plotArea.mouseDownTime;
+        var offset = (plotArea.mouseX - plotArea.mouseDownX) / cycleWidth;
+        plotArea.scrollAcc = offset * frameInterval / time;
+    },
```

Comment:
without these lines, whenever I clicked on the pause button then the **time scale numbers** and **waves** got vanished. So I had to add these lines to avoid that bug.

Reply:
I believe it should be 
```
this.autoScroll = false;
plotArea.scrollAcc = 0;
```